### ISSUE TITLE
Page Load Error

### ISSUE THREAD

**Issue by grahampcurran** [hudec117/sf-user-perm-report#32]
Describe the bug:
When in Production instance I receive the following error in the Chrome console
`build.js:2 Uncaught (in promise) TypeError: Cannot read property 'textContent' of undefined
    at Function.merge (build.js:2)
    at i.runReport (build.js:2)
    at chrome-extension://gkibdicghcpcikhjpgjoijpobdipklnp/public/async chrome-extension:/gkibdicghcpcikhjpgjoijpobdipklnp/build.js:2
DevTools failed to load SourceMap: Could not load content for chrome-extension://gkibdicghcpcikhjpgjoijpobdipklnp/build.js.map: HTTP error: status code 404, net::ERR_UNKNOWN_URL_SCHEME
`

To Reproduce Steps to reproduce the behavior:

Go to your Production instance Setup > Users > Select a User
Click on the Open Permission Report button
See error in Chrome Dev Console
Expected behavior: The Open Permission Report to load. Note; works as expected in sandbox environment for the same client.

Desktop (please complete the following information):

Browser version [e.g. 88.0.4324.182]
Extension version [e.g. 1.1.1
[gkibdicghcpcikhjpgjoijpobdipklnp-1614006214486.log](https://github.com/hudec117/sf-user-perm-report/files/6022867/gkibdicghcpcikhjpgjoijpobdipklnp-1614006214486.log)
]

**Comment by hudec117** [comment]
Thank you for reporting the error. Could you please perform this query:

```sql
SELECT PermissionSet.Name, PermissionSet.Type, PermissionSet.IsCustom, PermissionSet.NamespacePrefix FROM PermissionSetAssignment WHERE AssigneeId = 'USERID'
```
(replacing USERID with the user's ID)

**Feel free to censor PermissionSet.Name if it's non-standard.** If the user has a standard profile, it would be useful to know which one.

Obviously, given this is production please don't feel the need the share anything that may be sensitive.

Thank you!

**Comment by grahampcurran** [comment]
Hey!
See attached. All custom!
![2021-02-23 09_44_52-Window](https://user-images.githubusercontent.com/3749471/108826247-fa15d100-75bb-11eb-9a6f-2cedc5c2c497.png)

**Comment by hudec117** [comment]
Thank you!

It's likely the ones with "Group" and "Profile" permission set types that are causing the issue - for whatever reason SOQL won't allow you to filter on the "Type" field.

If I could replicate your user's setup that would be most helpful. Is the "Group" one a public group? A queue? What about the "Profile" one, is it actually a profile or is it that random-looking text?

**Comment by grahampcurran** [comment]
The Group one is a Permission Set Group containing several permission sets.
The Profile one is X00eb00000015uGGAAY which is the ID of a custom profile
with a leading X which is pretty interesting!

**Comment by grahampcurran** [comment]
The Group one is a Permission Set Group containing several permission sets.
The Profile one is X00eb00000015uGGAAY which is the ID of a custom profile with a leading X which is pretty interesting!

**Comment by hudec117** [comment]
Ah! I haven't considered permission set groups, I'll implement support for this.
I'll try and get the profile ones filtered out because they're not useful.

Thanks for all the info @grahampcurran, I should be able to get this fixed up.

**Comment by hudec117** [comment]
Fix implemented and it will be in the next 1.2.0 release. This is a larger release so it may be a couple of days until it is available on the Chrome Web Store.

**Comment by grahampcurran** [comment]
Awesome @hudec117 . Love the plugin. Well done!
